lib/phut/cli.rb

```diff
@@ -0,0 +1,55 @@
+require 'rake'
+require 'phut/settings'
+
+module Phut
+  class Cli
+    include FileUtils
+
+    def initialize(host)
+      @host = host
+    end
+
+    def send_packets(dest, options = {})
+      sh("#{executable} -i #{@host.interface} send_packets " \
+         "--ip_src #{@host.ip} --ip_dst #{dest.ip} " +
+         send_packets_options(options))
+    end
+
+    def show_tx_stats
+      puts stats(:tx)
+    end
+
+    def show_rx_stats
+      puts stats(:rx)
+    end
+
+    def add_arp_entry(other)
+      sh "sudo #{executable} -i #{@host.interface} add_arp_entry --ip_addr #{other.ip} --mac_addr #{other.mac}"
+    end
+
+    def set_ip_and_mac_address
+      sh "sudo #{executable} -i #{@host.interface} set_host_addr --ip_addr #{@host.ip} --ip_mask #{@host.netmask} --mac_addr #{@host.mac}"
+    end
+
+    private
+
+    def executable
+      "#{Phut::ROOT}/vendor/phost/src/cli"
+    end
+
+    def send_packets_options(options)
+      [
+       '--tp_src 1',
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.